storefront/app/helpers/spree/fonts_helper.rb

```diff
@@ -39,7 +39,7 @@ def google_fonts_link_tag
       return if fonts.blank?
 
       imports = fonts.map do |font|
-        "family=#{font.split(' ').join('+')}"
+        "family=#{font.split(' ').join('+')}:wght@200..700"
```

Comment:
```suggestion
        "family=#{font.split(' ').join('+')}:wght@100..900"
```

an even wider range :)